picoquic/picoquic.vcxproj

```diff
@@ -166,8 +166,6 @@
     <ClInclude Include="bytestream.h" />
     <ClInclude Include="democlient.h" />
     <ClInclude Include="demoserver.h" />
-    <ClInclude Include="fnv1a.h" />
```

Comment:
Was this intentional?

I like having the headers inside the solution as well to support "search in files" for these headers.

Reply:
fnv1a was removed from the solution. Democlient and Demoserver were moved to a different project. But removing bytestream is a mistake, thanks for noticing.

Reply:
Actually, I should also remove democlient and demoserver, since they are not part of the main picoquic project.

Reply:
Will leave fnv1a here for now, deal with removal later.